pages/common/mise.md

```diff
@@ -0,0 +1,32 @@
+# mise
+
+> Command-line interface for managing versions of different packages.
```

Comment:
```suggestion
> Manage versions of different packages.
```
We prefer to not mention terms like "Common-line" at the start of definitions as by default we document command line commands only (if it's a GUI or TUI in that case we mention it in the description).